### PR TITLE
IOSXE/show device tracking database int <interface>

### PR DESCRIPTION
**Pull request by robschn**
## Description
Adds parser for `show device tracking database int <interface>`

## Motivation and Context
Turns raw output of `show device tracking database int <interface>` into structured data.

## Impact (If any)
None

## Screenshots:
image

## Checklist:
- [X] I have updated the changelog.
- [X] I have updated the documentation (If applicable).
- [X] I have added tests to cover my changes (If applicable).
- [X] All new and existing tests passed.
- [X] All new code passed compilation.

### CODE REVIEW

**Comment by robschn** [comment]
PR opened before approval. Reopening now!

**Comment by robschn** [comment]
@tahigash let me know what you think! I used the `network_layer_address` as keys

File: src/genie/libs/parser/iosxe/tests/ShowDeviceTrackingDatabaseInt/cli/equal/golden_output_expected.py [hunk 1]
```diff
@@ -0,0 +1,836 @@
+expected_output = {
+    "binding_table": {"entries": 87, "dynamic": 75},
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/ShowDeviceTrackingDatabaseInt/cli/equal/golden_output_expected.py:2]: please add `'limit': 100000`.
```suggestion
    "binding_table": {"entries": 87, "dynamic": 75, "limit": 100000},
```

**Review by tahigash — CHANGES_REQUESTED** [review]

**Comment by tahigash** [comment]
> @tahigash let me know what you think! I used the `network_layer_address` as keys but that can be easily changed

`network_layer_address` key looks good. reviewed schema and left 1 comment for schema.

File: src/genie/libs/parser/iosxe/tests/ShowDeviceTrackingDatabaseInt/cli/equal/golden_output_expected.py (hunk 1 above)
Comment (robschn) [reply src/genie/libs/parser/iosxe/tests/ShowDeviceTrackingDatabaseInt/cli/equal/golden_output_expected.py]: Done!

File: src/genie/libs/parser/iosxe/show_device_tracking.py [hunk 2]
```diff
@@ -163,4 +163,122 @@ def cli(self, output=None):
         return device_tracking_database_dict
 
 
+# ======================================
+# Schema for:
+#  * 'show device-tracking database int'
+# ======================================
+class ShowDeviceTrackingDatabaseIntSchema(MetaParser):
+    """Schema for show device-tracking database int."""
```
Comment (choudnik) [inline src/genie/libs/parser/iosxe/show_device_tracking.py:171]: Can you please add  {interface} in command?

File: src/genie/libs/parser/iosxe/show_device_tracking.py [hunk 3]
```diff
@@ -163,4 +163,122 @@ def cli(self, output=None):
...
+                Optional("time_left"): str,
+                "vlan": int,
+            }
+        },
+    }
+
+
+# ======================================
+# Parser for:
+#  * 'show device-tracking database int'
```
Comment (choudnik) [inline src/genie/libs/parser/iosxe/show_device_tracking.py:192]: Can you please add {interface} in command?

**Review by choudnik — CHANGES_REQUESTED** [review]

File: changelog/undistributed/changelog_show_device-tracking_database_int_20201014194519.rst [hunk 4]
```diff
@@ -0,0 +1,6 @@
+--------------------------------------------------------------------------------
+                                New
+--------------------------------------------------------------------------------
+* IOSXE
+  * Added ShowDeviceTrackingDatabaseInt:
+      * show device-tracking database int
```
Comment (choudnik) [inline changelog/undistributed/changelog_show_device-tracking_database_int_20201014194519.rst:6]: show device-tracking database int {interface}

**Review by choudnik — CHANGES_REQUESTED** [review]

File: src/genie/libs/parser/iosxe/show_device_tracking.py (hunk 2 above)
Comment (robschn) [reply src/genie/libs/parser/iosxe/show_device_tracking.py]: Done!

File: changelog/undistributed/changelog_show_device-tracking_database_int_20201014194519.rst (hunk 4 above)
Comment (robschn) [reply changelog/undistributed/changelog_show_device-tracking_database_int_20201014194519.rst]: Done!

File: src/genie/libs/parser/iosxe/show_device_tracking.py (hunk 3 above)
Comment (robschn) [reply src/genie/libs/parser/iosxe/show_device_tracking.py]: Done!

**Review by choudnik — APPROVED** [review]

**Review by tahigash — APPROVED** [review]
schema looks good now

**Pull request merged**